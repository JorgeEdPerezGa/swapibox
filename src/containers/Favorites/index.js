import React, { Component } from 'react';

const Favorites = ({ card, handleFavorites }) => {
  return (
    <button
      onClick = {() => handleFavorites(card) }
      className = 'favorite-button'>favorite</button>
  );
};

export default Favorites;
