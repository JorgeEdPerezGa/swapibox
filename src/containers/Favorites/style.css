.card-list-favorites {
  padding-top: 80px; }

h5 {
  margin: 0;
  font-size: 30px;
  padding-top: 300px;
  color: white; }
