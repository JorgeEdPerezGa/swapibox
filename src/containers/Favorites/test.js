import React from 'react';
import ReactDOM from 'react-dom';
import Favorites from './index';
import setupTest from '../../setupTests';
import { shallow } from 'enzyme';

describe('<favorites/>', () => {
  const wrapper = shallow(<Favorites type = { [] }/>);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
