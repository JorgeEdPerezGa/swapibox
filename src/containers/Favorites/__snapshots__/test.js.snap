// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<favorites/> should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Favorites
    type={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h5>
        no favorites
</h5>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "no favorites",
      },
      "ref": null,
      "rendered": "no favorites",
      "type": "h5",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h5>
          no favorites
</h5>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "no favorites",
        },
        "ref": null,
        "rendered": "no favorites",
        "type": "h5",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
